src/index.css

```diff
@@ -1,3 +1,146 @@
-* {
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap');
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  padding: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  height: 100%;
+  width: 100vw;
+}
+
+#root {
+  width: 100%;
+  height: 100%;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 1px 5px 10px #343435;
+  padding: 10px;
+}
+
+nav {
+  margin-right: 10px;
+}
+
+nav ul {
+  display: flex;
+  list-style-type: none;
+  gap: 20px;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+nav ul a {
+  font-size: 27px;
+  text-decoration: none;
+  color: #0f0f11;
+}
+
+nav ul a:hover {
+  color: #000;
+  opacity: 0.6;
+}
+
+.active {
+  color: #343435;
+  font-weight: bold;
+  font-size: 40px;
+}
+
+.heading {
+  text-align: center;
+  font-size: 40px;
+  text-transform: capitalize;
+}
+
+.text {
+  padding: 20px;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+}
+
+.quote-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
While checking through this style.css file, I realized that you have used the `display: flex`, `justify-content: center` and `align-items: center` properties more than two times. I appreciate that the user interface styles are fine but **it is also a good programming practice to keep your code clean by not repeating particular lines of code multiple times (code duplication) instead we try to find a way to write clean, maintainable or reusable codes**. I kindly recommend that you create a class and add properties that are going to be needed multiple times. Then, you will add that class's name on every element that should possess those properties. For example; you can have the class `flex-container` with those three properties and then add it to all elements that should be flex and aligned in the center